portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro

```diff
@@ -27,6 +33,21 @@
 		<execute function="AssertTextEquals#assertPartialText" locator1="AppManager#APP_NAME" value1="${appName}" />
 	</command>
 
+	<command name="viewNoAppCP">
+		<var name="key_appName" value="${appName}" />
+
+		<execute function="AssertElementNotPresent" locator1="AppManager#APP_NAME" value1="${appName}" />
+
+		<while>
+			<condition function="IsElementNotPresent" locator1="Pagination#NEXT_LINK_DISABLED" />
+			<then>
+				<execute function="Click" locator1="Pagination#NEXT_LINK" />
+
+				<execute function="AssertElementNotPresent" locator1="AppManager#APP_NAME" value1="${appName}" />
+			</then>
+		</while>
```

Comment:
Let's emulate [this while loop](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro#L5-L10) to avoid introducing another locator.

Also we should refine the `Pagination#NEXT_LINK` locator (if it's not tedious), so that it doesn't also locate disabled "next" links. This way the while loop in this macro will end after the last enabled "next" link is clicked.

This path works for App Manager, but has not been tested elsewhere.
`
<tr>
	<td>NEXT_LINK</td>
	<td>//ul[contains(@class,'pagination')]/li[not(@class='disabled')]
/a[contains(.,'Next')] | //ul[contains(@class,'pagination')]/li [not(@class='disabled')]/a/span[contains(@class,'icon-caret-right')]</td>
	<td></td>
</tr>
`